tf2onnx/optimizer/const_fold_optimizer.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+
+"""const fold Optimizer.
+   if op's inputs are all const then do op computation when building the graph to improve performance
+   for example, input of transpose node is const then we can do transpose statically instead of at runtime
+"""
+
+import logging
+
+from tf2onnx.optimizer.optimizer_base import GraphOptimizerBase
+from tf2onnx import utils
+
+# pylint: disable=logging-not-lazy,unused-argument,missing-docstring
+
+_func_map = {}
+
+
+def _register_func(onnx_op):
```

Comment:
may need to consider domain and version as well, const fold logic may differ among different domains and versions.

Reply:
will add domain check.